src/components/typography/typography.component.js

```diff
@@ -166,10 +171,14 @@ const Typography = styled.span.attrs(
     line-height: ${lineHeight};
     margin: ${defaultMargin};
     padding: 0;
+    white-space: ${truncate ? "nowrap" : whiteSpace};
+    word-wrap: ${wordWrap};
+    text-overflow: ${truncate ? "ellipsis" : textOverflow};
+    overflow: ${truncate ? "hidden" : overflow};
```

Comment:
I guess we shouldn't provide overflow prop. It will allow users to put `overflow: scroll` and we don't want to let them do that.
I suggest to do
```js
${truncate && css`overflow: hidden`}
```

Reply:
Valid point. I'll get that changed to your suggestion 😃